sdks/python/src/honcho/peer.py

```
@@ -0,0 +1,311 @@
+            Response string containing the answer to the query, or None if no
+            relevant information is available
+        """
+        response = self._client.workspaces.peers.chat(
+            peer_id=self.id,
+            workspace_id=self.workspace_id,
+            queries=queries,
+            stream=stream,
+            target=str(target.id) if isinstance(target, Peer) else target,
+            session_id=session_id,
+        )
+        if response.content == "" or response.content == "None":
```

Pylint (3.3.7)

[refactor] Consider merging these comparisons with 'in' by using 'response.content in ('', 'None')'. Use a set instead if elements are hashable.

(R1714)